djangocon_europe/settings.py

```diff
@@ -278,3 +278,5 @@
 META_FB_TYPE = 'Article'
 META_GPLUS_SCOPE_CATEGORY = 'Article'
 BLOG_TWITTER_AUTHOR = '@DjangoConEurope'
+
+CFP_CLOSING_DATE = env('CFP_CLOSING_DATE')
```

Comment:
A Default must be provided here. I actually don't think we need this from the environment. Can be easily hardcoded in settings